test/tf/azure-vmss/modules/vmss/main.tf

```diff
@@ -11,7 +11,7 @@ resource "azurerm_public_ip" "test" {
   location                     = "${var.location}"
   resource_group_name          = "${var.resource_group}"
   public_ip_address_allocation = "static"
-  domain_name_label            = "${var.resource_group}"
+  domain_name_label            = "${random_string.domain_name.result}"
```

Comment:
Nice! I would do something like:

```
"${var.resource_group}-${random_string.domain_name.result}"
```

Just will be easier to debug then _just_ the random values.